### DISCUSSION TITLE
Steps to add a website/service so it can be scrobbled

### DISCUSSION THREAD

**Discussion by rohitsathish** [trakt-tools/universal-trakt-scrobbler#254] [Q&A]
Hi, this extension is great but its missing some of the services that I usually use to watch tv shows. Could someone explain to me, in a beginner-friendly way, how I can add more websites. For example - https://goplay.ml/? Thanks.

**Comment by MrMamen** [comment]
Sorry for late reply, but see the readme-file: https://github.com/trakt-tools/universal-trakt-scrobbler#how-to-add-scrobblersync-to-streaming-services

**Reply by dochypermoose** [reply]
Hey there, and it may be because I'm a 100% beginner, but I went to the location linked and that wasn't exactly "beginner-friendly" — I don't even know how to set up an API much less the remainder of the steps. Are there any plans to add Hulu as a streaming service?

**Reply by MrMamen** [reply]
I can agree that there are quite a few difficult steps, and the guide could probably be better. 
I would still try to run this:
npx trakt-tools dev create-service

Scribbling is easier to support than history sync, so I would start with that. Even if you can't figure out the code, all research could be helpful. Just write them on #260.
Important steps to figure out is;
- Is it possible to find an unique ID or code for each item when playing a title?
- can you see any lookups in the browser network tab (dev tools) for this id? If so what is the url and data?
- is there a history screen for watched items? If so can you find the url?

**Reply by dochypermoose** [reply]
I guess my first question is "where am I even entering in this code" 😅

**Reply by MrMamen** [reply]
Right. It requires some programming skills. You have to clone the git repo onto your computer, and run the command in the terminal. If these things are too difficult I would advise you first figuring out the questions I asked in the previous post. it will be helpful anyways.
